py3status/py3.py

```diff
@@ -84,6 +95,18 @@ def __init__(self, module=None, i3s_config=None, py3status=None):
                 self._i3s_config = config
             self._py3status_module = module.module_class
 
+    class Exceptions(Exception):
```

Comment:
How about `Py3Exception`?  I was initially thinking of something along those lines, but I had the whole name in mind and didn't want to be repetitive (`Py3.Py3StatusException`).  I'm thinking that this may be expanded into other types of exceptions down the road.

I had something like this in mind:
```
Py3
 +-- Py3Exception
      +-- SomeError
      |    +-- SomeNestedError
      +-- AnotherError
      +-- StatusError
      +-- HTTPError
           +-- URLError
           +-- HttpClientError
           +-- HttpServerError
```